src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maximumNumber = 100;
```

Comment:
here is how to create constants correctly: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names